src/Core/Localization/Currency.php

```diff
@@ -88,8 +88,6 @@ class Currency implements CurrencyInterface
 
     /**
      * Number of decimal digits to use with this currency.
-     *
-     * @var int
```

Comment:
Why?

Reply:
The lack of typehint is in cause here
class A {
/**
* @var int
**/
private $toto;
public function __construct($value) {
   $this->toto = $value;
}
public function doThis()
{
    return is_numeric($this->toto) ? 'plop' : 'paf';
}
}


you can pass whatever you like to $value, PHP won't complain but PHPStan will complain about your condition because $toto may be an integer

Reply:
I know, but the solution is to cast it when putting the value into ` toto`  instead :thinking:

Reply:
Or using ` int|null`

Reply:
I approve @PierreRambaud. For fixing it, typehint the constructor.

Reply:
suddently typehinting values introduced regressions more than once